tests/core/RecoverOperation.spec.ts

```diff
@@ -74,7 +74,15 @@ describe('RecoverOperation', async () => {
         unusedNextUpdateCommitmentHash
       );
 
-      (recoverOperationRequest.recovery_reveal_value as any) = 123; // Intentionally incorrect type.
+      const signedDataPayloadObject = {
+        delta_hash: 'deltaHash',
+        recovery_reveal_value: 123,
+        recovery_key: newRecoveryPublicKey,
+        recovery_commitment: unusedNextRecoveryCommitmentHash
+      };
+      const signedData = await OperationGenerator.signUsingEs256k(signedDataPayloadObject, recoveryPrivateKey);
+
+      recoverOperationRequest.signed_data = signedData; // Intentionally incorrect type for reveal value.
```

Comment:
Since we are just testing for the error thrown which got moved into , consider just do what "should throw if signedData contains an additional unknown property." does? ie. Move this test under `parseSignedDataPayload()` and invoke `parseSignedDataPayload()` directly and bypass/remove the additional setup code earlier. Same goes to other occurences.